app/jenkins-for-jira-ui/src/common/util/jenkinsConnectionsUtils.ts

```diff
@@ -12,19 +12,26 @@ export const setName = async (event: React.ChangeEvent<HTMLInputElement>, setSer
 	setServerName(event.target.value);
 };
 
+export const sanitizeInput = (input: string): string => {
+	// Replace <, >, &, ", ' with their HTML entities
+	return input
+		.replace(/</g, '&lt;')
+		.replace(/>/g, '&gt;')
+		.replace(/&/g, '&amp;')
+		.replace(/"/g, '&quot;')
+		.replace(/'/g, '&#39;');
+};
+
 // TODO kill off isFormValid when rollout is complete
 export const isValidServerName = (
 	value: string,
 	setErrorMessage: {
 		(value: SetStateAction<string>): void; (value: SetStateAction<string>): void; (arg0: string): void;
 	}
 ): boolean => {
-	if (!value) {
-		setErrorMessage('This field is required.');
```

Comment:
Removed this as this is only an error we show in the current flow. In the new flow, you can't click the next button if the input is empty.